.gulp/regeneration.iced

```diff
@@ -39,6 +42,12 @@ regenExpected = (opts,done) ->
     if (opts.flatteningThreshold)
       args.push("--#{opts.language}.payload-flattening-threshold=#{opts.flatteningThreshold}")
 
+    if (opts.generateMetadata)
+      args.push("--GenerateMetadata=true")
```

Comment:
suggest using the kebab-case-convention to match other autorest arguments

Reply:
Done.